contrib/rust-audit-info/template.py

```diff
@@ -0,0 +1,16 @@
+pkgname = "rust-audit-info"
+pkgver = "0.5.2"
+pkgrel = 0
+build_wrksrc = "rust-audit-info"
+build_style = "cargo"
+hostmakedepends = ["cargo"]
```

Comment:
you also need makedepends = ["rust-std"] in all of these

Reply:
which ones does "all of these" mean? Should I also open PRs for the ones you've already merged?

Reply:
yeah, i just missed those